models/palm_detection_mediapipe/mp_palmdet.py

```diff
@@ -68,17 +69,26 @@ def _postprocess(self, output_blob, original_shape):
         xy2 = (cxy_delta + wh_delta / 2 + self.anchors[:, :2]) * original_shape
         boxes = np.concatenate([xy1, xy2], axis=1)
         # NMS
-        keep_idx = cv.dnn.NMSBoxes(boxes, score, self.score_threshold, self.nms_threshold, top_k=1)
+        keep_idx = cv.dnn.NMSBoxes(boxes, score, self.score_threshold, self.nms_threshold, top_k=self.topK)
         if len(keep_idx) == 0:
-            return None, None, None
-        selected_score = score[keep_idx][0]
-        selected_box = boxes[keep_idx][0]
+            return None
```

Comment:
Returning an empty array would be better.

Reply:
After testing, there isn‘t much difference in speed on cpu before and after modification. But I found that the speed on RPI-CPU should be about 170ms instead of 89.41ms as written in the README. The time on intel-cpu is about 25ms instead of 15.5. Am I not configured correctly somewhere?